apb.yml

```diff
@@ -27,63 +27,64 @@ plans:
         required: True
         title: Sync Application Name
       - name: SYNC_APP_DOCKER_IMAGE
-        description: URL of your Sync Application Docker Image, e.g. docker.io/aerogear/voyager-server-example-task
+        description: URL of your GraphQL Server Docker Image, e.g. docker.io/aerogear/voyager-server-example-task
         type: string
         required: True
         title: Sync Application Docker image
       - name: SYNC_APP_DOCKER_IMAGE_VERSION
-        description: Version of your Sync Application Docker Image (using 'latest' if not specified)
+        description: (default value: 'latest') Version of your Sync Application Docker Image
         type: string
         required: False
         title: Sync Application Docker image version
+        default: 'latest'
       - name: SYNC_APP_HEALTH_ENDPOINT
-        description: Health Endpoint of your Sync Application (using '/' if not specified)
+        description: (default value: '/') Health Endpoint of your Sync Application. Use 'health' for docker.io/aerogear/voyager-server-example-task image.
```

Comment:
there is already a `/` in the template, should the default value here just be an empty string?

Reply:
As we discussed, we should also give user an option to not use the health endpoint at all and this is not possible atm (it requires some changes to provision role). I will address this in other PR.